operators/bosh-operator/DirectorService.js

```diff
@@ -237,18 +247,27 @@ class DirectorService extends BaseDirectorService {
     };
     return this
       .initialize(operation)
+      .catch(err => {
+        logger.error(`Error occurred while finding network index for update
```

Comment:
same comment as above with regards to logger.

Reply:
done